content/blog/how-to-set-your-development-enviroment/index.md

```diff
@@ -39,31 +42,24 @@ This guide will help you to set your development enviroment, follow this guide i
 
 ### Installation
 
-1. clone the OpenInfraMX's [repository](https://github.com/opensource-latinamerica/openinframx)
+1. Clone the OpenInfraMX's [repository](https://github.com/opensource-latinamerica/openinframx) and its submodules
 
 ```bash
-git clone https://github.com/opensource-latinamerica/openinframx
+git clone --recurse-submodules --depth 1 https://github.com/opensource-latinamerica/openinframx
 ```
 
-2. Add the necessary submodules to your
-
-```bash
-git submodule init
-git submodule update
-```
-
-3. Install npm dependencies
+2. Install npm dependencies
 
 ```bash
 npm install
 ```
 
-4. Start the server
+3. Start the server
 
 ```bash
 hugo server
 ```
 
 # Final Words
 
-If you followed the steps feel free to reach us at our [github](https://github.com/opensource-latinamerica/openinframx) and help in any way you want. It can be a fun ride
+If you followed the steps feel free to reach us at our [github](https://github.com/opensource-latinamerica/openinframx) and help in any way you want. It can be a fun ride.
```

Comment:
```suggestion
Any contribution is welcome, feel free to reach us through our [GitHub repository](https://github.com/opensource-latinamerica/openinframx). This could be an amazing journey.
```